documentation/asciidoc/computers/raspberry-pi/bcm2711-bootloader.adoc

```diff
@@ -559,6 +559,33 @@ This option may be set to 0 to block self-update without requiring the EEPROM co
 
 Default: `1`
 
+==== `secure-boot` configuration properties in `config.txt`
+The following `config.txt` properties are used to program the `secure-boot` OTP settings. These changes are irreversible and can only be programmed via `RPIBOOT` when flashing the bootloader EEPROM image.  This ensures that `secure-boot` cannot be set remotely or via accidentally inserting a stale SD card image.
+
+[[program_pubkey]]
+===== program_pubkey
+If this property is set to `1` then `recovery.bin` will write the hash of the public key in the EEPROM image to OTP.  Once set, the bootloader will reject EEPROM images signed with different RSA keys or unsigned images.
```

Comment:
Does this mean that you have to program the EEPROM image first and _then_ `program_pubkey`, or can both steps be done as part of the same process?

Reply:
It's part of the flashing process